cypress/integration/steps.spec.js

```diff
@@ -295,10 +301,10 @@ context('Steps', () => {
       cy.get('[data-test=step]').first().should('not.have.class', '-last');
     });
 
-    it('last step should not contain error', () => {
+    it('last step should not contain killed/skipped', () => {
       cy.get('[data-test=step]').last().as('echoStep');
       cy.get('@echoStep').should('be.visible').click({ force: true });
-      cy.get('@echoStep').should('not.contain', 'error:');
+      cy.get('@echoStep').should('not.contain', 'step was killed');
```

Comment:
i see we removed the previous check of to make sure it doesn't contain an error. I think we want to keep that check and add this new check on top of it, because it should be neither. you'll want to update the description of the test then as well.

Reply:
Added.